src/backupNg/backupNg.spec.js

```diff
@@ -232,4 +232,64 @@ describe("backupNg", () => {
       expect(vmTask.data.id).toBe(vmIdWithoutDisks);
     });
   });
+
+  test("execute two times a rolling snapshot with 2 as retention & revert to an old state", async () => {
+    jest.setTimeout(7e4);
+    const SR = await xo.getSrId();
+    const vmId = await xo.createTempVm({
+      name_label: "XO Test Temp",
+      name_description: "Creating a temporary vm",
+      template: config.xoTestTemplateId,
+      VDIs: [
+        {
+          device: "0",
+          size: 1,
+          SR,
+          type: "user",
+        },
+      ],
+    });
+
+    const scheduleTempId = randomId();
+    const { id: jobId } = await xo.createTempBackupNgJob({
+      ...defaultBackupNg,
+      vms: {
+        id: vmId,
+      },
+      schedules: {
+        [scheduleTempId]: {
+          name: "scheduleTest",
+          cron: "0 * * * * *",
+        },
+      },
+      settings: {
+        [scheduleTempId]: { snapshotRetention: 2 },
```

Comment:
`reportWhen` should be set to `never` to avoid conflicts with the plugin `Backup reports`